wolfcrypt/src/asn.c

```diff
@@ -36319,6 +36325,7 @@ int ParseCRL(RevokedCert* rcert, DecodedCRL* dcrl, const byte* buff, word32 sz,
         ret = PaseCRL_CheckSignature(dcrl, buff, cm);
     }
 
+    (void)verify;
```

Comment:
Already this line above WOLFSSL_MSG.

Reply:
Thanks for catching that. I opened the PR before that was added.